library/core/class.autoloader.php

```diff
@@ -898,6 +898,30 @@ protected function GetClassNameFromFile($File) {
    public function MapType() {
       return $this->MapInfo['maptype']; //GetValue('maptype', $this->MapInfo);
    }
+   
+   /**
+    * Fix path with backslash for Windows OS.
+    *
+    * @param string $path
+    * @return string
+    *
+    * Check to see if the path is not empty and if it is not in a form of drive: then replace any "\" with "/" to avoid 
+    * the parse_fn_file bug and it also further check to see if it has only one "/" and the OS is "WIN" then it will prepend "/" 
+    * in front of the path so it has the correct file path.
+    */
+   function fixBackSlash($path) {
+      if (!empty($path) && !preg_match('/[A-Za-z]\:/', $path)) {
```

Comment:
I discover this bug when I was doing testing with Vanilla that reside on a network share and Vanilla just not working so I start a debug session and found out that the network share path was chop off by parse_ini_file so that is when I come up with a fix.